packages/pn-personagiuridica-webapp/src/components/IntegrazioneApi/VirtualKeyContextMenu.tsx

```diff
@@ -56,41 +57,10 @@ const VirtualKeyContextMenu: React.FC<Props> = ({ data, keys, handleModalClick }
         key.user?.fiscalCode === data.user?.fiscalCode && key.status === VirtualKeyStatus.Blocked
     ) && data.status !== VirtualKeyStatus.Rotated;
 
-  return (
-    <Box data-testid="contextMenu">
-      <IconButton
-        onClick={handleClick}
-        size="small"
-        color="primary"
-        data-testid="contextMenuButton"
-        aria-label={t('context-menu.title')}
-        aria-controls={open ? 'context-menu' : undefined}
-        aria-haspopup="true"
-        aria-expanded={open ? 'true' : undefined}
-      >
-        <MoreVert />
-      </IconButton>
-
-      <Menu
-        data-testid="menuContext"
-        anchorEl={anchorEl}
-        open={open}
-        onClose={handleClose}
-        onClick={handleClose}
-        slotProps={{
-          paper: {
-            elevation: 0,
-            sx: {
-              overflow: 'visible',
-              filter: 'drop-shadow(0px 2px 8px rgba(0,0,0,0.32))',
-              mt: 1.5,
-            },
-          },
-        }}
-        transformOrigin={{ horizontal: 'right', vertical: 'top' }}
-        anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
-      >
-        {shouldShowRotateButton() && (
+    const handleMenu = () =>{
```

Comment:
Fix the indentation. I rather use the code currently inside the handleMenu directly without using the function call pushing the object into the menuElements array. You can extract the code contained into the handleMenu function, remove the handleMenu function and its call and place the extracted code directly here.